apps/nestjs-wallet/docs/tutorial.md

```diff
@@ -1,5 +1,56 @@
 # Tutorial
 
+## Business overview
+The business objective of this tutorial is to demonstrate how to issue a proof in the form of a Verifiable Credential (VC) to a known user connected to a portal that we will call "authority portal".
+In the example below, we will issue a permanent residency card to a user we will call "citizen". We use the context "https://w3id.org/citizenship/v1" which describes the data of a VC of type "PermanentResidentCard".
+
+Here is an extract of the context:
+
+        ...
+        "id": "@id",
+        "type": "@type",
+
+        "ctzn": "https://w3id.org/citizenship#",
+        "schema": "http://schema.org/",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+        "birthCountry": "ctzn:birthCountry",
+        "birthDate": {"@id": "schema:birthDate", "@type": "xsd:dateTime"},
+        "commuterClassification": "ctzn:commuterClassification",
+        "familyName": "schema:familyName",
+        "gender": "schema:gender",
+        "givenName": "schema:givenName",
+        "lprCategory": "ctzn:lprCategory",
+        "lprNumber": "ctzn:lprNumber",
+        "residentSince": {"@id": "ctzn:residentSince", "@type": "xsd:dateTime"}
+        ...
+
+To get the definition of the field "gender" which points to the definition "schema:gender", use the URI of "schema" which points to "http://schema.org/", so we will have a definition available at "http://schema.org/gender".
+TODO interpretation of id and type.
```

Comment:
I'm not sure what you mean by "interpretation of id and type". 
I think probably you are looking for an explanation of why these two lines are in the `context`:
```
"id": "@id",
"type": "@type",
```
If so, the short answer is that is they are aliasing `@id` and `@type` to `id` and `type` so that they are compatible with `json`. (At least I think that's the answer 😄 )

Reply:
Ho do you know that type can take one of the https://w3id.org/citizenship/v1 values? If PermanentResident and Person where at a lower level than PermanentResidentCard, that could be understandable.

Reply:
Hmm, well I think `type` is a generic concept from JSON-LD (https://json-ld.org/spec/latest/json-ld/#specifying-the-type) and the VC data model seems to allow any URI for the type. https://www.w3.org/TR/vc-data-model/#types
So I'm not sure there's a way to unambiguously classify a URI as being suitable to be a VC type.
Does that answer your question?

Reply:
```suggestion
Regarding the aliasing of `id` to `@id` and `type` to `@type`, see the [VC specification](https://www.w3.org/TR/vc-data-model/#syntactic-sugar).
```